addons-example.yml

```
@@ -37,6 +36,10 @@ my-s3-bucket:
     production:
       bucket_name: my-bucket-prod
       deletion_policy: Retain
+      retention_policy: # Optional
+        mode: GOVERNANCE # or COMPLIANCE
```

What is the difference? Is that documented somewhere?

---

Governance means the bucket objects can be modified by an admin with the appropriate perms. Compliance means not even admins can remove or modify objects in the bucket.

---

FYI: https://aws.amazon.com/blogs/storage/protecting-data-with-amazon-s3-object-lock/

---

I think this might be helpful...
```suggestion
        mode: GOVERNANCE # or COMPLIANCE (See https://aws.amazon.com/blogs/storage/protecting-data-with-amazon-s3-object-lock/)
```